packages/react-instantsearch-dom-geo/src/GoogleMaps.js

```diff
@@ -23,6 +24,10 @@ class GoogleMaps extends Component {
     }),
   };
 
+  isUserInteraction = true;
```

Comment:
The map could be updated by the user or programmatically. But the events are triggered in both cases. When we trigger programmatic events we need to "lock" the map to avoid to trigger event listeners.